src/ngraph/op/interpolate.hpp

```diff
@@ -17,38 +17,132 @@
 #pragma once
 
 #include "ngraph/op/op.hpp"
+#include "ngraph/op/util/attr_types.hpp"
 
 namespace ngraph
 {
     namespace op
     {
-        typedef struct
-        {
-            AxisSet axes;
-            std::string mode;
-            bool align_corners = true;
-            bool antialias = false;
-            std::vector<size_t> pads_begin;
-            std::vector<size_t> pads_end;
-        } InterpolateAttrs;
-
         namespace v0
         {
+            /// \brief Structure that specifies attributes for interpolation
+            struct InterpolateAttrs
+            {
+                // specify dimension indices where interpolation is applied, and `axes` is any
+                // unordered list of indeces of different dimensions of input tensor. Required.
+                AxisSet axes;
+                // specifies type of interpolation
+                // one of `nearest`, `linear`, `cubic`, `area`. Required.
+                std::string mode;
+                // a flag that specifies whether to align corners or not.
+                // `true` (default) means the alignment is applied,
+                // `false` means the alignment isn't applied.
+                bool align_corners = true;
+                // a flag that specifies whether to perform anti-aliasing. default is `false`
+                bool antialias = false;
+                // specify the number of pixels to add to the beginning of the image being
+                // interpolated. This addition of pixels is done before interpolation calculation.
+                std::vector<size_t> pads_begin;
+                // specify the number of pixels to add to the end of the image being interpolated.
+                // This addition of pixels is done before interpolation calculation.
+                std::vector<size_t> pads_end;
+            };
+
             /// \brief Layer which performs bilinear interpolation
             class NGRAPH_API Interpolate : public Op
             {
             public:
+                enum class InterpolateMode
+                {
```

Comment:
Make the enum case the same as in the spec, i.e. "nearest" instead of "NEAREST".

Reply:
Changed the enums to lowercase.